build/scripts/apiserver.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+usage="$0 start|stop"
+if [ $# -ne 1  ]; then
+    echo USAGE: $usage
+    exit 1
+fi
+
+case $1 in
+start)
+    rm -f /tmp/apiserver-fifo
+
+    set -e
+    echo starting apiserver
+    /usr/bin/contiv-vol-run.sh apiserver
+
+    # now just sleep to keep the service up
+    mkfifo "/tmp/apiserver-fifo"
+    < "/tmp/apiserver-fifo"
+    ;;
+
+stop)
+    echo stopping apiserver
+    rm -f /tmp/apiserver-fifo
+    docker stop apiserver
+    docker rm -v apiserver
```

Comment:
why are we removing volumes here? this doesn't remove the container I don't think.